packages/documentation/copy/en/handbook-v2/Classes.md

```diff
@@ -887,7 +887,7 @@ class S {
 
 ### Why No Static Classes?
 
-TypeScript (and JavaScript) don't have a construct called `static class` the same way as, for example, C# does.

```

Comment:
I don't think that this change is correct; the wording here is saying "JavaScript doesn't have static classes like C# does", which implies that C# supports static classes.